network/p2p/discover_test.go

```diff
@@ -512,37 +518,109 @@ func Test_GetNodesForDiscover(t *testing.T) {
 func Test_AddNewList(t *testing.T) {
 	dis := newDiscover()
 	assert.NoError(t, dis.Start())
+	dis.deputyNodes = make(map[common.Hash]*RawNode, 20)
+	dis.foundNodes = make(map[common.Hash]*RawNode, 20)
+	dis.whiteNodes = make(map[common.Hash]*RawNode, 20)
+
+	for i := 0; i <= 5; i++ {
```

Comment:
很丑。应该封装一个函数，把需要变化的部分传进去。一共才6个节点，在这里调用6次就可以了。如
```suggestion
	setupRawNode(dis.whiteNodes, table[0], -1, 10)
	setupRawNode(dis.deputyNodes, table[1], -2, 11)
	...
```